### PR TITLE
Modify UserGuide.md to include checkout feature

### LINKED ISSUE #178: Update UserGuide.md to include checkout feature
UserGuide.md does not contain details about the new checkout feature. 
It should be updated to include those details for the user.

### PR DESCRIPTION
**Pull request by deeyonn**
Resolves #178

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -226,39 +242,35 @@ Examples:
...
 
 
-### 3.9 Deleting an applicant: `delete`
+### 3.10 Check-out a new or existing list: `checkout`
 
-Deletes the specified applicant from InternConnect.
+Switches between different lists in InternConnect.
 
-Format: `delete INDEX`
+Format: `checkout file_name`
```
Comment (cliftonfelix) [inline docs/UserGuide.md:249]: Please use CAPITAL letters here

File: docs/UserGuide.md [hunk 2]
```diff
@@ -270,16 +282,17 @@ If your changes to the data file makes its format invalid, InternConnect will di
...
+| **Find**     | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                                                                                                                                        |
+| **Import**   | `import FILE_PATH`<br> e.g., `import nus_students.json`                                                                                                                                                                                                                           | 
+| **Export**   | `export`                                                                                                                                                                                                                                                                          |
+| **Checkout** | `checkout 27-oct-2022`                                                                                                                                                                                                                                                            |
```
Comment (cliftonfelix) [inline docs/UserGuide.md:296]: To standardize the format, please use `checkout FILE_NAME` <be> e.g., `checkout 27-oct-2022`

**Review by lulucopter — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (deeyonn) [reply docs/UserGuide.md:249]: Line changed to `Format: `checkout FILE_NAME``

File: docs/UserGuide.md (hunk 2 above)
Comment (deeyonn) [reply docs/UserGuide.md:296]: Line changed to `| **Checkout** | checkout FILE_NAME `

File: docs/UserGuide.md [hunk 3]
```diff
@@ -246,7 +246,7 @@ Examples:
 
 Switches between different lists in InternConnect.
 
-Format: `checkout file_name`
+Format: `checkout FILE_NAME`
 
 * New and existing lists will be stored at `data/file_name.json`
```
Comment (cliftonfelix) [inline docs/UserGuide.md:251]: FILE_NAME here

File: docs/UserGuide.md [hunk 4]
```diff
@@ -270,16 +282,17 @@ If your changes to the data file makes its format invalid, InternConnect will di
...
+| **Find**     | `find KEYWORD [MORE_KEYWORDS]...`<br> e.g., `find James Jake`                                                                                                                                                                                                                     |
+| **Import**   | `import FILE_PATH`<br> e.g., `import nus_students.json`                                                                                                                                                                                                                           | 
+| **Export**   | `export`                                                                                                                                                                                                                                                                          |
+| **Checkout** | `checkout FILE_NAME`                                                                                                                                                                                                                                                              |
```
Comment (cliftonfelix) [inline docs/UserGuide.md:296]: Please put the  e.g., checkout 27-oct-2022 also

File: docs/UserGuide.md (hunk 3 above)
Comment (deeyonn) [reply docs/UserGuide.md:251]: Line changed to `* New and existing lists will be stored at data/FILE_NAME.json`

**Review by cliftonfelix — APPROVED** [review]
LGTM! Thanks for the changes

**Pull request merged**